hab-pkg-deb/bin/hab-pkg-deb.sh

```diff
@@ -0,0 +1,443 @@
+    print_help
+    exit_with "You must specify a Habitat package." 1
+  fi
+
+  pkg_install_path=$(hab pkg path "$pkg")
+
+  #
+  # If *inst or *rm scripts are included with the package, use them.
+  #
+  if [[ -z "$preinst" ]] && [[ -e "$pkg_install_path/bin/preinst" ]]; then
+    preinst="$pkg_install_path/bin/preinst"
+  fi
+
+  if [[ -z "$postinst" ]] && [[ -e "$pkg_install_path/bin/postinst" ]]; then
+    postinst="$pkg_install_path/bin/postinst"
+  fi
+
+  if [[ -z "$prerm" ]] && [[ -e "$pkg_install_path/bin/prerm" ]]; then
+    prerm="$pkg_install_path/bin/prerm"
+  fi
+
+  if [[ -z "$postrm" ]] && [[ -e "$pkg_install_path/bin/postrm" ]]; then
+    postrm="$pkg_install_path/bin/postrm"
+  fi
+}
+
+# The name converted to all lowercase to be compatible with Debian naming
+# conventions
+convert_name() {
+  if [[ ! -z "$debname" ]]; then
+    safe_name="${debname,,}"
+  else
+    safe_name="${pkg_origin,,}-${pkg_name,,}"
+  fi
+}
+
+# Return the Debian-ready version, replacing all dashes (-) with tildes
+# (~) and converting any invalid characters to underscores (_).
+convert_version() {
+  if [[ $pkg_version == *"-"* ]]; then
+    safe_version="${pkg_version//-/\~}"
+    warn "Dashes hold special significance in the Debian package versions. "
+    warn "Versions that contain a dash and should be considered an earlier "
+    warn "version (e.g. pre-releases) may actually be ordered as later "
+    warn "(e.g. 12.0.0-rc.6 > 12.0.0). We'll work around this by replacing "
+    warn "dashes (-) with tildes (~). Converting '$pkg_version' "
+    warn "to '$safe_version'."
+	else
+    safe_version="$pkg_version"
+	fi
+}
+
+description() {
+  pkg_description="$(head -2 <<< "$manifest" | tail -1)"
+
+  # TODO: Handle multi-line descriptions.
+  # See https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Description
+  # Handle empty pkg_description
+  if [[ -z "$pkg_description" ]]; then
+    if [[ ! -z "$debname" ]]; then
+      echo "$debname"
+    else
+      echo "$pkg_name"
+    fi
+  else
+    echo "$pkg_description"
+  fi
+}
+
+maintainer() {
+  pkg_maintainer="$(grep __Maintainer__: <<< "$manifest" | cut -d ":" -f2 | sed 's/^ *//g')"
+
+  if [[ -z "$pkg_maintainer" ]]; then
+    echo "$pkg_origin"
+  else
+    echo "$pkg_maintainer"
+  fi
+}
+
+# Output the contents of the "control" file
+render_control_file() {
+  hab pkg exec core/handlebars-cmd handlebars \
+    --pkg_name "$safe_name" \
+    --pkg_version "$safe_version" \
+    --pkg_license "$pkg_license" \
+    --pkg_origin "$pkg_origin" \
+    --architecture "$(architecture)" \
+    --maintainer "$(maintainer)" \
+    --installed_size "$(installed_size)" \
+    --section "$(section)" \
+    --priority "$(priority)" \
+    --pkg_upstream_url "$pkg_upstream_url" \
+    --description "$(description)" \
+    --conflicts "$conflicts" \
+    --depends "$depends" \
+    --provides "$provides" \
+    --replaces "$replaces" \
+    < control/control.hbs \
```

Comment:
Can an end user can provide there own `control/control.hbs`? Can this be provided at `export/deb/control.hbs`? We should also drop the `*.hbs` on this file. Idiomatically we don't use an extension like this on any other Handlebars templates in Habitat.